dependencies.yaml

```diff
@@ -0,0 +1,209 @@
+  - dask/label/dev
+  - conda-forge
+  - nvidia
+dependencies:
+  common_build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cmake>=3.23.1,!=3.25.0
+      - output_types: conda
+        packages:
+          - libcumlprims=23.02.*
+          - libraft-headers=23.02.*
+          - libraft-distance=23.02.*
+          - libraft-nn=23.02.*
+          - rmm=23.02.*
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              arch: x86_64
+            packages:
+              - gcc_linux-64=9.*
+              - sysroot_linux-64==2.17
+          - matrix:
+              arch: aarch64
+            packages:
+              - gcc_linux-aarch64=9.*
+              - sysroot_linux-aarch64==2.17
+  cpp_build:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              arch: x86_64
+              cuda: "11.5"
+            packages:
+              - nvcc_linux-64=11.5
+          - matrix:
+              arch: aarch64
+              cuda: "11.5"
+            packages:
+              - nvcc_linux-aarch64=11.5
+          - matrix:
+              arch: x86_64
+              cuda: "11.4"
+            packages:
+              - nvcc_linux-64=11.4
+          - matrix:
+              arch: aarch64
+              cuda: "11.4"
+            packages:
+              - nvcc_linux-aarch64=11.4
+          - matrix:
+              arch: x86_64
+              cuda: "11.2"
+            packages:
+              - nvcc_linux-64=11.2
+          - matrix:
+              arch: aarch64
+              cuda: "11.2"
+            packages:
+              - nvcc_linux-aarch64=11.2
+          - matrix:
+              arch: x86_64
+              cuda: "11.0"
+            packages:
+              - nvcc_linux-64=11.0
+          - matrix:
+              arch: aarch64
+              cuda: "11.0"
+            packages:
+              - nvcc_linux-aarch64=11.0
+  py_build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cuda-python>=11.7.1,<12.0
+          - scikit-build>=0.13.1
+          - cython>=0.29,<0.30
+          - treelite=3.0.1
+      - output_types: conda
+        packages:
+          - cudf=23.02.*
+          - nccl>=2.9.9
+          - raft-dask=23.02.*
+          - pylibraft=23.02.*
+  py_run:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - dask>=2022.9.2
+          - distributed>=2022.9.2
+          - joblib>=0.11
+      - output_types: conda
+        packages:
+          - cupy>=7.8.0,<12.0.0a0
+          - dask-cudf=23.02.*
+          - dask-cuda=23.02.*
+          - ucx>=1.13.0
+          - ucx-py=0.29.*
+          - ucx-proc=*=gpu
+    specific:
+      - output_types: requirements
+        matrices:
+          - matrix:
+              arch: x86_64
+            packages:
+              - cupy-cuda115>=7.8.0,<12.0.0a0 # TODO: This might change to cupy-cuda11x?
```

Comment:
Do we really support cupy versions that old? (7.8.0 released on Aug 19, 2020) We require 9.5.0 in cudf.

https://github.com/rapidsai/cudf/blob/b24e1fa04ab9c18fc132535b5dbddca639608911/dependencies.yaml#L245
```suggestion
              - cupy-cuda115>=9.5.0,<12.0.0a0 # TODO: This might change to cupy-cuda11x?
```